src/citrine/resources/project_roles.py

```diff
@@ -0,0 +1,5 @@
+from typing import Union
+
+LEAD = "LEADs"
```

Comment:
Is this a typo (the "s" at the end of "LEADs")?

Reply:
yes